.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+# Recommend matching the black line length (default 88),
+# rather than using the flake8 default of 79:
+max-line-length = 88
```

Comment:
@tiberiuichim don't we set this in buildout already? I usually set max-line-length to 200. Otherwise, it can interfere with black.

Reply:
@tisto It's for the integration with my vim setup, I've installed [flake8-black](https://pypi.org/project/flake8-black/) (I usually use autopep8). They recommend adding this file to the project. The problem with buildout settings is that it's not read by the editor setup.

Reply:
I am fine with moving the buildout settings to a flake8 config file. Though, we should use the 200 line-length then.